userspace/libsinsp/plugin.cpp

```diff
@@ -136,6 +133,13 @@ sinsp_plugin::~sinsp_plugin()
 {
 	destroy();
 	plugin_unload(m_handle);
+	
+	auto cur_async_handler = m_async_evt_handler.load();
+	if (cur_async_handler)
+	{
+		delete cur_async_handler;
+		m_async_evt_handler.store(nullptr);
```

Comment:
Shouldn't we first store nullptr (so that others threads loading it will get nullptr) and then delete the pointer?
Otherwise another thread could `load` the handler after we removed it but before we stored nullptr.